src/cfnlint/core.py

```diff
@@ -104,10 +104,11 @@ def create_parser():
 
     # Alllow the template to be passes as an optional or a positional argument
     standard.add_argument(
-        'template', nargs='?', help='The CloudFormation template to be linted')
+        'template', nargs='*', help='The CloudFormation template to be linted')
```

Comment:
This is a real nitpick, but should probably call this variable 'templates' (with a metavar of TEMPLATE).